hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -50,17 +50,17 @@
     {
       name: data_toggle_clear
       desc: '''
-            Verify the data toggle clear functionality.
+            Verify the data toggle restore functionality.
```

Comment:
It's a bit confusing that this describes `data_toggle_clear`, rather than `data_toggle_restore` (which is below)!

Would it make sense to extend this sentence to make it look less the testpoint with this description should be called `data_toggle_restore`?